documentation/src/main/asciidoc/userguide/chapters/query/hql/QueryLanguage.adoc

```diff
@@ -761,8 +761,8 @@ Let's see what these functions do.
 [[hql-function-type]]
 ===== `type()`
 
-The function `type()`, applied to an identification variable, evaluates to the entity name of the referenced entity.
-This is mainly useful when dealing with entity inheritance hierarchies.
+The function `type()`, applied to an identification variable, evaluates to the type name of the referenced entity or embeddable.
```

Comment:
In fact this is wrong. In HQL the `type()` function evaluates to a `Class`, not a type name.

Reply:
Nice catch, thanks!